collections/_release-notes/2025-08-05-1-205-0.md

```diff
@@ -0,0 +1,14 @@
+---
+title: 08.05.2025
+layout: productupdates
+tags:  bugfix sdk 
+date: 2025-08-05
+
+---
+
+Today’s release includes the following updates:
+
+- Support ellipsis use in SDK sandbox environment
+- Change MANIFEST.json default fields (non-breaking)
+- Fix a bug with external identifier event firing too often
```

Comment:
We should specify this is SDK events and the event name specifically

Reply:
fun fact, it doesn't look like this event made it into the documentation in the first place (it was from a community contributor) so I will remove this (since they are the only one aware that it was added and isn't working as expected) and pick up an action item to add to the documentation instead!